playwright.config.ts

```diff
@@ -78,7 +78,7 @@ export default defineConfig({
     command: "npm run build && cd .output && node ./server/index.mjs",
     url: "http://127.0.0.1:3000",
     timeout: 120 * 1000,
-    reuseExistingServer: !process.env.CI,
+    reuseExistingServer: true,
```

Comment:
I think that setting this to true on CI as well as locally means that it should use the app running in docker which was spun up in a preverious step of the action.

Reply:
If that were true, wouldn't it mean that we should not run the command on line 78, because it would be duplicating running the app?

Reply:
I think it only runs the command if there's no existing server - so it's still useful to keep that for local dev. 

From https://playwright.dev/docs/test-webserver
> reuseExistingServer: If true, it will re-use an existing server on the url when available. If no server is running on that url, it will run the command to start a new server.